benchadapt/python/benchadapt/result.py

```diff
@@ -210,6 +214,48 @@ def to_publishable_dict(self):
         return res_dict
 
 
+def validate_or_remove_github_commit_key(res_dict: Dict, strict=False):
+    """
+    Mutate BenchmarkResult dictionary in-place to make its `github` key
+    property be compliant with the Conbench HTTP API.
+
+    Not providing the `github` key in result dictionary tells Conbench that
+    this result is commit context-less (recording it in Conbench might be
```

Comment:
```suggestion
    this result is commit-context-less (recording it in Conbench might be
```

Super minor, but we're talking about the commit-context here yeah?

Reply:
Yes, thanks! :) Hyphenation is a difficult topic sometimes, and I am glad you gave feedback here!